sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -262,6 +264,9 @@ internal class BitmapWidgetRenderer(
     this.bitmap = bitmap
     this.bitmapWidth = bitmap.width
     this.bitmapHeight = bitmap.height
+    this.halfBitmapWidth = bitmapWidth / 2f
+    this.halfBitmapHeight = bitmapHeight / 2f
```

Comment:
I think you still need to update the `translateMatrix` after the bitmap size has changed.
Which also creates some challenges if the user has previously called `setTranslation` and expects those deltas to be kept after changing the bitmap

Reply:
I agree it should be called here as `leftX()` and `topY()` are dependent on bitmap size

I don't see an an issue if `setTranslation()` was called beforehand, should be fine imo

Reply:
Also dont see an issue with updating translation after bitmap change, seems should be enough to add `setTranslation(this.translationX, this.translationY)` to update the matrix accordingly